style.css

```diff
@@ -0,0 +1,832 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.navbar {
+  display: none;
+}
+
+.navbar1 {
+  width: 100vw;
+  height: 30px;
+  background-color: #272a31;
+  color: white;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.navbar1 ul {
+  display: flex;
+  list-style: none;
+  margin-right: 15%;
+}
+
+.navbar1 ul li,
+.navbar2 ul li {
+  margin-left: 10px;
+}
+
+.navbar2 a {
+  color: #272a31;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.navbar2-menu a {
+  color: #272a31;
```

Comment:
- [ ] Please write DRY CSS. You have multiple instances of the rule `color:#272131` in your code base. Please write one class with this rule and assign it to all elements that need to have a color of #272a31.  DRY code is easy to maintain and your end up with less code 👨🏾‍💻 
- You have 20 instances of this rule. ✍🏾  Please refactor your code and have only one.

Reply:
Hello Francis,

Done.  Thanks